src/files/files-list/FilesList.js

```diff
@@ -33,6 +36,7 @@ export class FilesList extends React.Component {
     updateSorting: PropTypes.func.isRequired,
     root: PropTypes.string.isRequired,
     downloadProgress: PropTypes.number,
+    filesIsFetching: PropTypes.boolean,
```

Comment:
```suggestion
    filesIsFetching: PropTypes.bool,
```
My bad, i think i added that.